site/docs/v1/tech/admin-guide/upgrade.adoc

```diff
@@ -4,44 +4,300 @@ title: Upgrade FusionAuth
+** <<Skipping Versions>>
+** <<Migrating Configuration Between Environments>>
+** <<Themes and Customizations>>
+* <<Upgrade Options>>
+** <<Easiest Route>>
+** <<Rolling Upgrade>>
+** <<Blue-Green Deployment>>
+** <<Out-of-Band Database Upgrades>>
+* <<FAQs>>
+* <<Detailed Upgrade Instructions>>
+** <<Docker>>
+** <<Homebrew>>
+** <<ZIP Packages>>
+** <<Linux Packages>>
+** <<FastPath>>
 
-If you used the FastPath or other installation methods initially, you may prefer to use that same mode to download the latest version of FusionAuth. If you would like to continue by manually downloading the necessary packages, read on.
+== When to Upgrade
+
+=== Reasons To Upgrade
+
+Upgrading to a newer version of FusionAuth can provide the following benefits:
+
+* Access to new features and improvements.
+* Improved performance and stability.
+* Critical security updates and patches.
+
+=== Reasons Not To Upgrade
+
+You might want to delay upgrading in the following situations:
+
+* Your current version meets all your requirements.
+* You are in the middle of a project or have tight deadlines.
+* You have concerns about compatibility with customizations or third-party integrations.
+
+=== Being Aware of Upgrade Options
+
+You can stay informed about new FusionAuth releases and updates by:
+
+* Monitoring the link:/docs/v1/tech/release-notes[release notes].
+* Joining the https://fusionauth.io/community[FusionAuth community] forum and/or slack.
+* Following FusionAuth on social media platforms, including https://twitter.com/FusionAuth[Twitter] and https://www.linkedin.com/company/fusionauth[LinkedIn].
+* Monitoring the https://github.com/FusionAuth/fusionauth-issues/issues[FusionAuth Issues GitHub repository].
+
+== How to Think About Upgrades
+
+Before upgrading FusionAuth, there are a number of things to consider. Upgrading FusionAuth is similar to upgrading a library your application depends on. Go through each of the points below to see if any of them apply to your situation.
+
+=== Update Contents and Notes
+
+Read the link:/docs/v1/tech/release-notes[release notes]. Release notes help you understand how an update affects you. They include details on new features, improvements, bug fixes, and potential breaking changes.
+
+=== Testing
+
+Before starting the upgrade process on production, test the upgrade in a staging or development environment to identify potential issues. This is particularly important on more complex upgrades involving multiple version jumps or if you have made customizations to FusionAuth.
+
+=== Downtime and Database Migrations
+
+Plan for any downtime or maintenance windows required for the upgrade. If changes to the database schema are required, this could require more steps and downtime that affects end users. Such changes are always noted in the release notes. (Read the release notes carefully.) A pure application upgrade, on the other hand, may not require any downtime from an end-user perspective.
```

Comment:
```suggestion
Plan for any downtime or maintenance windows required for the upgrade. If changes to the database schema are required, this could require more steps and downtime that affects end users. Such changes are always noted in the release notes. (Read the release notes carefully.) An upgrade without a schema change, on the other hand, may not require any downtime from an end-user perspective.
```